frontend/src/components/ContributionMessages/ContributionMessagesListItem.vue

```diff
@@ -1,19 +1,24 @@
 <template>
   <div class="contribution-messages-list-item">
-    <is-not-moderator v-if="isNotModerator" :message="message"></is-not-moderator>
-    <is-moderator v-else :message="message"></is-moderator>
+    <div v-if="isNotModerator" class="is-not-moderator text-right">
+      <b-avatar :text="initialLetters" variant="info"></b-avatar>
+      <span class="ml-2 mr-2">{{ message.userFirstName }} {{ message.userLastName }}</span>
+      <span class="ml-2">{{ $d(new Date(message.createdAt), 'short') }}</span>
+      <div class="mt-2">{{ message.message }}</div>
+    </div>
+    <div v-else class="is-moderator text-left">
+      <b-avatar square :text="initialLetters" variant="warning"></b-avatar>
+      <span class="ml-2 mr-2">{{ message.userFirstName }} {{ message.userLastName }}</span>
+      <span class="ml-2">{{ $d(new Date(message.createdAt), 'short') }}</span>
+      <small class="ml-4 text-success">{{ $t('community.moderator') }}</small>
+      <div class="mt-2">{{ message.message }}</div>
+    </div>
   </div>
 </template>
-<script>
-import IsModerator from '@/components/ContributionMessages/slots/IsModerator.vue'
-import IsNotModerator from '@/components/ContributionMessages/slots/IsNotModerator.vue'
 
+<script>
 export default {
   name: 'ContributionMessagesListItem',
-  components: {
-    IsModerator,
-    IsNotModerator,
-  },
   props: {
     message: {
```

Comment:
same same as above 👆🏼

Reply:
This has also been fixed by [d37b8d3](https://github.com/gradido/gradido/commit/d37b8d3e50ab9e87512c76e59b646ab097b0bd82).